Create Pull Request Template (#324)
MicrosoftDocs/WindowsCommunityToolkitDocs: .github/PULL_REQUEST_TEMPLATE.md

<!-- When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `live` branches:

For documentation for new features, please base your fork off the master branch.

If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [live branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/live).  This will allow us to get the change to the published documentation between releases.

We will periodically merge updates from the live branch to master to keep master in-sync with the published docs.  When we make a new release, we will push master to the live branch in order to publish documentation for new features.

Documentation Links
**This link is currently only available for Microsoft Employees** - [Staging review from 'master' branch](https://review.docs.microsoft.com/windows/communitytoolkit/?branch=master)
- [Live site from 'live' branch](https://docs.microsoft.com/windows/communitytoolkit) -->

> ## Fixes # <!-- Link to relevant issue or Feature PR of the Windows Community Toolkit repo (for ex: #1234) which will create a reference to the associated issue and PR once it is created-->

Reviewer: ```suggestion
## Fixes # <!-- Link to a relevant issue # in this docs repository (if any, otherwise remove this line) -->
## Docs for Toolkit PR [#](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/#) <!-- Link to relevant issue or Feature PR # of the Windows Community Toolkit repo which will create a reference to the associated issue and PR once it is created, remove if not tied to an issue or feature -->
```

<!-- When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `live` branches:

For documentation for new features, please base your fork off the master branch.

If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [live branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/live).  This will allow us to get the change to the published documentation between releases.

We will periodically merge updates from the live branch to master to keep master in-sync with the published docs.  When we make a new release, we will push master to the live branch in order to publish documentation for new features.

Documentation Links

> - [Staging review from 'master' branch](https://review.docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/?branch=master)

Reviewer: ```suggestion
- [Staging review from 'master' branch](https://review.docs.microsoft.com/windows/communitytoolkit/?branch=master)
```
We should call out this link is unfortunately currently only available as a Microsoft employee. Maybe we just keep/update this info in the repository readme vs. in the PR template?

Author: Updated link with the instruciton

> - [Live site from 'live' branch](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit) -->

Reviewer: `uwpcommunitytoolkit` was the old name, this is the new link:
```suggestion
- [Live site from 'live' branch](https://docs.microsoft.com/windows/communitytoolkit) -->
```
We also should strip any of the locale from the links, the doc site will automatically redirect to the proper locale (even though our docs are English, I believe the rest of the navigation can be localized)

Author: Updated link

> ## Fixes # <!-- Link to relevant issue (for ex: #1234) which will create reference to the associated issue once the PR is created. -->

Reviewer: We should call out here to that we should also link to the feature PR in the Windows Community Toolkit repo

<!-- Add a brief overview here of the feature/bug & fix. -->

> ## PR Type

Reviewer: Think we can drop this section as this is the docs repo :)

Author: Dropped!

What kind of change does this PR introduce?
<!-- Please uncomment one or more that apply to this PR. -->

<!-- - Bugfix -->
<!-- - Feature -->
<!-- - Code style update (formatting) -->
<!-- - Refactoring (no functional changes, no api changes) -->
<!-- - Build or CI related changes -->
<!-- - Documentation content changes -->
<!-- - Other... Please describe: -->

> ## What is the current behavior?

Reviewer: Merge these two sections to "What changes to the docs does this PR provide?"

Author: Merged!

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

## What is the new behavior?
<!-- Describe how was this issue resolved or changed? -->

## PR Checklist

Please check if your PR fulfills the following requirements:

> - [ ] Tested code with current [supported SDKs](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/blob/master/docs/toc.md#controls)

Reviewer: Think this checklist is 
- [ ] Correctly picked the right branch to base the change off (`master` for new features, `live` for typos/improvements)
- [ ] For new pages, used the [provided template](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/blob/master/docs/.template.md)
- [ ] For new features, added an entry in the [Table of Contents](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/blob/master/docs/toc.md)
- [ ] Ran against a spell checker

Thoughts?

Author: Just added grammar and the rest looks good.
